scripts/transform-images.js

```diff
@@ -0,0 +1,37 @@
+import { existsSync } from 'fs';
```

Comment:
why do we need this, instead of re-using the vite optimizer?

Reply:
This is not for smaller image outputs, but rather, for the assets page, where there's the option to download the same image in multiple formats.